static/app/views/issueDetails/traceTimeline/useTraceTimelineEvents.tsx

```diff
@@ -7,21 +7,24 @@ import {useApiQuery} from 'sentry/utils/queryClient';
 import useOrganization from 'sentry/utils/useOrganization';
 
 interface BaseEvent {
+  culprit: string; // Used for subtitle
   id: string;
   'issue.id': number;
-  message: string;
   project: string;
   'project.name': string;
   timestamp: string;
   title: string;
   transaction: string;
 }
 
-interface TimelineDiscoverEvent extends BaseEvent {}
```

Comment:
This was incorrect as it should have contained `event.type` and `stack.function`.